TicTacToeApp/GamePrompt.swift

```diff
@@ -1,29 +1,28 @@
 
+import UIKit
+
 public class GamePrompt {
+  var prompt: UILabel
 
-  public init() {
-    // initialize with the UILabel?
+  public init(prompt: UILabel) {
+    self.prompt = prompt
```

Comment:
I remember that this suggestion of injecting the `UILabel` was one we less certain on than others. Do you feel it improved the design?